radio/util/fwoptions.py

```diff
@@ -125,8 +125,8 @@
 options_gruvin9x = {
     "heli": ("HELI", "YES", "NO"),
     "templates": ("TEMPLATES", "YES", "NO"),
-    "nofp": ("FLIGHT_MODES", "NO", "YES"),
     "nocurves": ("CURVES", "NO", "YES"),
+    "nofp": ("FLIGHT_MODES", "NO", "YES"),
```

Comment:
"nofm" would be better (for 2.3 ...)

Reply:
no even, those are 'unsupported' boards options :D